### PR TITLE
Update stable demo

### PR DESCRIPTION
**Pull request by TheMilek**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -55,3 +45,10 @@
 ###> lexik/jwt-authentication-bundle ###
 /config/jwt/*.pem
 ###< lexik/jwt-authentication-bundle ###
+
+###> symfony/webpack-encore-bundle ###
+/node_modules/
```
Comment (coldic3) [inline .gitignore:50]: It's already ignored in line 12.

File: .env.test [hunk 2]
```diff
@@ -12,3 +12,7 @@ JWT_PASSPHRASE=ALL_THAT_IS_GOLD_DOES_NOT_GLITTER_NOT_ALL_THOSE_WHO_WANDER_ARE_LO
 # Sync transport turned for testing env for the ease of testing
 MESSENGER_TRANSPORT_DSN=sync://
 ###< symfony/messenger ###
+
+###> symfony/mailer ###
+MAILER_DSN=null://null
+###< symfony/mailer ###
```
Comment (coldic3) [inline .env.test:18]: No empty line at the end.

File: assets/admin/entry.js [hunk 3]
```diff
@@ -0,0 +1 @@
+import 'sylius/bundle/AdminBundle/Resources/private/entry';
```
Comment (coldic3) [inline assets/admin/entry.js:1]: I guess we should follow this approach just to be consistent everywhere: https://github.com/Sylius/Sylius-Standard/commit/e97cb7c578ac071ab6cb4438b252be1dbd85abe5

File: config/packages/assets.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+framework:
+  assets:
+    packages:
+      admin:
+        json_manifest_path: '%kernel.project_dir%/public/build/admin/manifest.json'
+      shop:
+        json_manifest_path: '%kernel.project_dir%/public/build/shop/manifest.json'
```
Comment (GSadee) [inline config/packages/assets.yaml:7]: Use 4 spaces as an indentation

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Kamil! :1st_place_medal:

**Pull request merged**